src/pages/components/button/usage.mdx

```diff
@@ -280,36 +275,39 @@ button locations include:
 </Column>
 </Row>
 
-| Alignment         | Use case                                                                                                                                                        |
-| ----------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| _Left-justified_  | Banner call to actions, in-page forms, and nested buttons in components like tiles                                                                              |
-| _Right-justified_ | Inline notifications, inline field buttons and data tables, progressive forms, wizards, and single-button dialogs                                               |
-| _Full-span_       | Dialogs, side panels, and small tiles; currently Carbon does not offer a way to implement full-span buttons in code, without an override, they max out at 320px |
+<Caption>Alignment of buttons across various layouts.</Caption>
 
-#### Fluid versus default buttons
+| Alignment       | Use case                                                                                                                                                        |
+| --------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Left-justified  | Banner call to actions, in-page forms, and nested buttons in components like tiles.                                                                             |
+| Right-justified | Inline notifications, inline field buttons and data tables, progressive forms, wizards, and single-button dialogs.                                              |
+| Full-span       | Dialogs, side panel, and small tiles; currently Carbon does not offer a way to implement full-span buttons in code, without an override, they max out at 320px. |
```

Comment:
Is this part still true? Did we check with a dev? "currently Carbon does not offer a way to implement full-span buttons in code, without an override, they max out at 320px"

```suggestion
| Full-span       | Dialogs, side panel, and small tiles; currently Carbon does not offer a way to implement full-span buttons in code, without an override, they max out at 320px. |
```